<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-06-11T16:59:40 ICT" name="Default suite" finished-at="2025-06-11T16:59:53 ICT" duration-ms="12817">
    <groups>
      <group name="regression">
        <method signature="firstclass.SecondtMethod()[pri:0, instance:firstpackage.firstclass@3cebbb30]" name="SecondtMethod" class="firstpackage.firstclass"/>
      </group> <!-- regression -->
      <group name="smoke">
        <method signature="firstclass.FirstMethod()[pri:0, instance:firstpackage.firstclass@3cebbb30]" name="FirstMethod" class="firstpackage.firstclass"/>
      </group> <!-- smoke -->
    </groups>
    <test started-at="2025-06-11T16:59:40 ICT" name="Default test" finished-at="2025-06-11T16:59:53 ICT" duration-ms="12817">
      <class name="firstpackage.firstclass">
        <test-method signature="FirstMethod()[pri:0, instance:firstpackage.firstclass@3cebbb30]" started-at="2025-06-11T16:59:40 ICT" name="FirstMethod" finished-at="2025-06-11T16:59:53 ICT" duration-ms="12657" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FirstMethod -->
        <test-method signature="SecondtMethod()[pri:0, instance:firstpackage.firstclass@3cebbb30]" started-at="2025-06-11T16:59:53 ICT" name="SecondtMethod" finished-at="2025-06-11T16:59:53 ICT" duration-ms="29" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [lehoainam] but found [hoainam]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [lehoainam] but found [hoainam]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at firstpackage.firstclass.SecondtMethod(firstclass.java:34)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SecondtMethod -->
      </class> <!-- firstpackage.firstclass -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
